db/migrate/20190429143724_create_job_runs.rb

```diff
@@ -0,0 +1,14 @@
+class CreateJobRuns < ActiveRecord::Migration[5.1]
+  def change
+    create_table :job_runs do |t|
+      t.string :host
+      t.string :pid
+      t.datetime :start_time
+      t.datetime :finish_time
+      t.string :job_name
+      t.string :result
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
Dont forget to add index on pid or other column that will be frequently queried on.  Thinking job_name

Reply:
Indexes added and columns marked nullable/not nullable